tests/test_locales.py

```diff
@@ -997,72 +997,77 @@ def test_format_relative_future(self):
 
 @pytest.mark.usefixtures("lang_locale")
 class TestKoreanLocale:
-    def test_describe(self):
-        assert self.locale.describe("now", only_distance=True) == "방금"
-        assert self.locale.describe("now", only_distance=False) == "지금 막"
-
     def test_format_timeframe(self):
+        assert self.locale._format_timeframe("now", 0) == "지금"
+        assert self.locale._format_timeframe("second", 1) == "1초"
+        assert self.locale._format_timeframe("seconds", 2) == "2초"
+        assert self.locale._format_timeframe("minute", 1) == "1분"
+        assert self.locale._format_timeframe("minutes", 2) == "2분"
         assert self.locale._format_timeframe("hour", 1) == "한시간"
         assert self.locale._format_timeframe("hours", 2) == "2시간"
-
         assert self.locale._format_timeframe("day", 1) == "하루"
         assert self.locale._format_timeframe("days", 2) == "2일"
-
         assert self.locale._format_timeframe("week", 1) == "1주"
         assert self.locale._format_timeframe("weeks", 2) == "2주"
-
         assert self.locale._format_timeframe("month", 1) == "한달"
         assert self.locale._format_timeframe("months", 2) == "2개월"
-
         assert self.locale._format_timeframe("year", 1) == "1년"
         assert self.locale._format_timeframe("years", 2) == "2년"
 
-
     def test_format_relative_now(self):
-
-        result = self.locale._format_relative("조금 전", "now", 0)
-
-        assert result == "조금 전"
+        assert self.locale._format_relative("지금", "now", 0) == "지금"
 
     def test_format_relative_past(self):
```

Comment:
Hmm, wouldn't this be relative_future? we're using positive deltas here.

Reply:
Actually I've copied test method names(test_format_relative_past, test_format_relative_future) from other language's test code just above.
I'll merge codes in relative past and future format tests into one `test_format_relative'.